gateway/modules/auth/match.go

```diff
@@ -128,7 +128,7 @@ func (m *Module) matchOr(ctx context.Context, projectID string, rule *config.Rul
 		}
 	}
 	// if condition is not satisfied -> return empty model.PostProcess and error
-	return nil, ErrIncorrectMatch
+	return nil, formaterror(rule, ErrIncorrectMatch)
 }
 
 func match(rule *config.Rule, args map[string]interface{}) error {
```

Comment:
Dont return match errors from matchString, Match Number,etc as is. Pass it through format error